EnglishFilter/sections/general_extensions.txt

```diff
@@ -98,7 +98,7 @@ gaybeeg.info,italiashare.life#%#(function(){var b=window.setTimeout;window.setTi
 oladblock.me,oladblock.xyz,oladblock.services,oload.space,oload.live,oload.club,openload.pw,oload.fun,povwideo.cc,turbobit.cc,fruithosts.net,oload.icu,putlockermix.org,streamplay.me,empressleak.biz,xxxity.com,oload.win#%#Object.defineProperty(window, 'BetterJsPop', { get: function() { return { init: function() { } }; } });
 ! `#%#//scriptlet("abort-on-property-read", "BetterJsPop")`
-vizplay.*,view47.*,streamplusvip.*,oyohd.com,hindilinks4uto.com,animestream.watch,msubplix.com,vidxhot.net,waaw.tv,waaw.to,chillicams.net,flashx.net,hqq.to,justswallows.com,openload.casa,meucdn.vip,yandexcdn.com,firstr0w.eu,hqq.tv,onlystream.tv,nhentai.net,streamz.*,hindipix.in,3movs.com,alltube.tv,oload.monster,tryboobs.com,eporner.com,oload.press,vivatube.com,oload.life,streamp1ay.me,nxload.com,oload.services,verystream.com,freeteenporn.xxx,ashemale.one,kaotic.com,hentaicore.org,oload.vip,woof.tube,vidop.icu,oload.best,oload.network,alltube.pl,oload.biz,oload.website,oload.xyz#%#//scriptlet("abort-on-property-read", "BetterJsPop")
+glygrebedu.pro,autoizers.*,vizplay.*,view47.*,streamplusvip.*,oyohd.com,hindilinks4uto.com,animestream.watch,msubplix.com,vidxhot.net,waaw.tv,waaw.to,chillicams.net,flashx.net,hqq.to,justswallows.com,openload.casa,meucdn.vip,yandexcdn.com,firstr0w.eu,hqq.tv,onlystream.tv,nhentai.net,streamz.*,hindipix.in,3movs.com,alltube.tv,oload.monster,tryboobs.com,eporner.com,oload.press,vivatube.com,oload.life,streamp1ay.me,nxload.com,oload.services,verystream.com,freeteenporn.xxx,ashemale.one,kaotic.com,hentaicore.org,oload.vip,woof.tube,vidop.icu,oload.best,oload.network,alltube.pl,oload.biz,oload.website,oload.xyz#%#//scriptlet("abort-on-property-read", "BetterJsPop")
```

Comment:
The same question.

Reply:
As I said, check yourself, too much to explain

Reply:
[The same](https://github.com/AdguardTeam/AdguardFilters/pull/75051#discussion_r577474504).

Reply:
@zloyden 
![image](https://user-images.githubusercontent.com/76908410/108188101-28258c00-714a-11eb-920f-661e1db7d45d.png)

Reply:
As I said earlier, this scriptlet is for original domains, not the adservers. Also please use the spoiler for the screenshots.

Reply:
I do not know how. I have removed them.

Reply:
No problem, I edited your post. There is a code for the spoiler.

```
<details><summary>Screenshot:</summary>

![image](%screenshot_url%)
</details><br/>
```